backstop/scenarios/header.js

```diff
@@ -4,6 +4,11 @@ module.exports = [
   // Header types
   scenario('Type Default', {
     selectors: ['#demo-default'],
+    hideSelectors: ['.iui-urgent::before'],
```

Comment:
You shouldn't need to hide animated parts anymore as all animations are stopped in tests.

Reply:
Ah, you are correct.  Removed.